src/gui/tray/UserLine.qml

```diff
@@ -119,6 +131,13 @@ MenuItem {
                 icon.source: "qrc:///client/theme/more.svg"
                 icon.color: "transparent"
 
+                Accessible.role: Accessible.ButtonMenu
+                Accessible.name: qsTr("More")

```

Comment:
If this is the 3-dot menu in the account list, the name would be better as "Account actions". Also doesn’t need a description.